bridge/util.go

```diff
@@ -30,9 +30,13 @@ func combineTags(tagParts ...string) []string {
 	return tags
 }
 
-func serviceMetaData(env []string, port string) map[string]string {
+func serviceMetaData(config *dockerapi.Config, port string) map[string]string {
+	meta := config.Env
```

Comment:
Good catch. The likelihood of it actually causing a problem is very low IMO, but cleaning it up would be nice.